src/pages/postReservation/components/left_side/CardNotice.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { DiamondPinkIcon } from '../../../../assets/svgs';
+import { CardNoticeContent } from '../../constatnts/postResevationText.ts';
+
+type Props = {
+  hostName: string;
+};
+const CardNotice = (props: Props) => {
```

Comment:
옹 여기 interface가 아닌 type을 쓴 이유가 있나요 ? 

그리고 저희 prop 타입 네이밍 컨벤션!
```suggestion
interface CardNoticeProps = {
  hostName: string;
};
const CardNotice = (props: CardNoticeProps) => {
const {hostName} = props;
```
<img width="817" alt="스크린샷 2024-05-21 오후 6 13 01" src="https://github.com/NOW-SOPT-CDSP-TEAM-WEB2/Client/assets/109661444/2401d7c0-07c5-448d-9d1d-6d153fd17189">

Reply:
으악 제 실수입니다 ㅜ_ㅜ 바로 수정 들어가겠슴니다!!